### ISSUE TITLE
Binary release

### ISSUE THREAD

**Issue by veimox** [pal-robotics/pal_statistics#4]
Could these packages be released in the main distros as `kinetic` and `melodic`?

The only issue I see is that the package `pal_carbon_collector` depends on `python-graphitesend-pip` that is not installable as a debian package, but we can add it in the README or ignore that package for the release.

This would go along with: https://github.com/pal-robotics/statsdcc/issues/2

**Comment by v-lopez** [comment]
Most likely yes, let us look into it..

**Comment by v-lopez** [comment]
I believe it should work since I added the dependecy a while ago. 
https://github.com/ros/rosdistro/blob/master/rosdep/python.yaml#L1857

We cannot remove it from the package.xml because it's used in our internal build system.

**Comment by v-lopez** [comment]
I am afraid http://prerelease.ros.org/melodic is not working for me and I cannot generate a command to test if the release would go through.

I'll try later, or if someone can see if they can generate the command, I will test it.

**Comment by veimox** [comment]
The issue with pip packages is that they will not get installed as dependencies of the binary because APT cannot manage them. Thus for a complete binary, all dependencies should also should also be packed as debian. In this case is fairly simple because [the original package](https://github.com/daniellawrence/graphitesend) already contains the debian files to be binarized. Running `dpkg-buildpackage -b -us -uc` can generate the file.

The main issue is that the binary has to be put in the ROS repos, which means that you will have to create the release repo, and so on....

To be fair, from our perspective, we can deploy this internally so is not a showstopper for us, I just thought that it would be nice that the community could make use of these fantastic packages :smile: 

If you dont want to go through the process then is understandable that we close this issue

**Comment by v-lopez** [comment]
I believe the release will work because the rosdep key can be resolved using the python.yaml. The thing is that it won't install it, but as long as the packages compile, I'm totally fine with releasing it.

I just wanted to test it in the prerelease, but I can't generate the comamnd to test it on melodic or kinetic, from firefox or chrome.

**Comment by veimox** [comment]
Im ok with the release too as if any it seems better to use `statsdcc`...

I have ended up deploying these in our servers in `kinetic-devel` so if it helps, i can guarantee that it works there :smile:

I had problems with `pal_statsd_collector` that even though is installing `python-statsd` is using the library `pystatsd`. I have changed the code to use `python-statsd` and now it works fine. So far is a bit irrelevant becase is catkin-ignored...

Thanks for the package btw!
